azurerm/internal/services/apimanagement/api_management_resource.go

```diff
@@ -291,7 +300,53 @@ func resourceArmApiManagementService() *schema.Resource {
 							Default:  false,
 						},
 
-						"enable_triple_des_ciphers": {
```

Comment:
You can maintain compatibility by keeping the existing property, adding the `Deprecated` field, and checking for both in the `expandApiManagementCustomProperties()` function (preferring the new one).

Reply:
Added. Not sure if the `d.GetOk(...)` works for the nested block as expected though.